lib/matplotlib/axes/_base.py

```diff
@@ -566,11 +566,26 @@ def __init__(self, fig, rect,
         if self.yaxis is not None:
             self._ycid = self.yaxis.callbacks.connect('units finalize',
                                                       self.relim)
-        self.tick_params(top=rcParams['xtick.top'],
-                         bottom=rcParams['xtick.bottom'],
-                         left=rcParams['ytick.left'],
-                         right=rcParams['ytick.right'],
-                         which='both')
+
+        self.tick_params(top=rcParams['xtick.top'] and
+                         rcParams['xtick.minor.top'],
```

Comment:
Bottom is the longest, and should be 79 if moved to its own line. I mean like this:
```python
        self.tick_params(
            top=...
            bottom=...
            ...)
```